favorites.lisp

```diff
@@ -10,13 +10,13 @@
 (defvar *favorites/destroy* "https://api.twitter.com/1.1/favorites/destroy.json")
 (defvar *favorites/create* "https://api.twitter.com/1.1/favorites/create.json")
 
-(defun favorites/list (&key user-id screen-name (count 20) since-id max-id (include-entities T))
+(defun favorites/list (&key user-id screen-name (count 20) since-id max-id (include-entities T) (tweet_mode "extended"))
```

Comment:
Do not use underscores in argument names.
Also do not use strings as values that should be of a limited set. Use keywords instead.